packages/elements/src/pagination/index.ts

```diff
@@ -231,62 +276,34 @@ export class Pagination extends BasicElement {
 
   /**
    * Check a new page value to be usable
-   * if a new page value is allow then return newPage
+   * if a new page value is allow then return a new
    * Condition to be old value is null or NaN or undefined or string or less than 1
-   * @param oldPage a old page value
-   * @param newPage a new page value
+   * @param oldValue a old page value
+   * @param newValue a new page value
    * @return return a new page value
    */
-  private validatePage (oldPage: string, newPage: string): string {
-    let page = Number.parseInt(newPage, 10);
+  private validatePage (oldValue: string, newValue: string): string {
```

Comment:
Create a validator for both input change and value setter.

Reply:
Already fixed